keps/sig-release/1731-publishing-packages/README.md

```diff
@@ -149,51 +183,236 @@ Scenario: [...]
 ```
 -->
 
-### Implementation Details/Notes/Constraints
+### Risks and Mitigations
 
-Packages will be published for different:
+- _Risk_: The OBS installation provided by openSUSE is unable to serve the load generated by the Kubernetes project
```

Comment:
> openSUSE is unable to serve the load generated by the Kubernetes project

Do we have the metrics for this?

I'm mainly curious about the load OBS can serve, by default, and potential mitigation size.

Reply:
> Do we have the metrics for this?

What type of metrics? If you think about our current load, we don't have any information about that. 

> I'm mainly curious about the load OBS can serve, by default, and potential mitigation size.

@darix @dirkmueller Can you please provide some feedback on this?

Reply:
I think this is unrelated to the "OBS installation" but to the Download infrastructure (which is separate to the Building part, it is the publishing aspect of it). The Download infrastructure is a geo replicated mirrors located in various places on the planet. We can scale it if there is need to do so and the kubernetes project is able to help with resources. 

Also, if there is a potential concern about the Download infrastructure, then this part can be entirely replaced with something entirely different serving only the kubernetes files only for the kubernetes project. So there shouldn't be the concern of lock-in in this aspect. as long as there is a way for the OBS publishing service to be able to "stage" new releases on your download site, we'd be able to do so.

Reply:
This will be documented in the latest KEP update.

Reply:
As Dirk mentioned we do not have to use our opensuse.org download infrastructure for your projects. we can publish directly to your download infrastructure and you handle that part yourself.